README.md

```diff
@@ -0,0 +1,144 @@
+<div align="center">
+  <h1>Blog App</h1>
+</div>
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Blog App ](#-blog-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blog App <a name="about-project"></a>
+
+> This is a blog app based on ruby on rails. This app will feature users, posts, comments, likes.
+
+## 🛠 Built With <a name="built-with"></a>
+<h2> Tech Stack </h2>
+- Ruby on Rails
+- PSQL
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Display Users
+
+- Display posts
+
+- Display comments on posts
+
+- Display likes on posts
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Example command:
+
+```sh
+ Install Ruby
+ Install rails
+```
```

Comment:
- [OPTIONAL] Great work creating the README as directed by the previous reviewer. For this command, it is not `Install Ruby` or `Install rails` but `bundle install`. I would advise you to correct this as proper instructions help people who clone your repository know how to run it locally. 👍 😎

Reply:
Thank you & noted